errbot/backends/telegram_messenger.py

```diff
@@ -369,6 +416,12 @@ def _telegram_special_message(self, chat_id, content, msg_type, **kwargs):
                                              latitude=kwargs.pop('latitude', ''),
                                              longitude=kwargs.pop('longitude', ''),
                                              **kwargs)
+        elif msg_type == 'contact':
```

Comment:
oh, sorry, okay. it's not exactly linked to file transfer, it works similarly to location and venue. they are just metadata. the API functionality is similar though. I will remove this, for a next PR.